src/magicbox/SuggestionsManager.ts

```diff
@@ -185,6 +200,7 @@ export class SuggestionsManager {
 
     if (!this.hasSuggestions) {
       this.removeAccessibilityPropertiesForSuggestions();
+      $$(this.root).trigger(OmniboxEvents.querySuggestLoseFocus);
```

Comment:
Could the name of the event be too specific to our current use-case? Would a more general description of the event be `focusedQuerySuggestPreviewChanged` or `activeQuerySuggestPreviewChanged`?

It's a slight change, but I am thinking of a professional services person who might one day want to subscribe to this event.

Reply:
This will also trigger when exiting and not going on other querySuggest

Personally, with the name `activeQuerySuggestPreviewChanged`, I would think that we would have a `NEW` query suggestion selected but it is not always the case.

I have another event: `querySuggestGetFocus` I could rename this one if you wish. On that case, we would always have a new querySuggestion selected